install_config/configuring_aws.adoc

```diff
@@ -18,6 +18,21 @@ volumes as persistent storage] for application data. After AWS is configured
 properly, some additional configurations will need to be completed on the
 OpenShift hosts.
 
+[IMPORTANT]
+====
+Switching from not using a cloud provider to using a cloud provider produces an
+error message. Adding the cloud provider tries to delete the node because the
+node switched from using the *hostname* as the `*externalID*` (which would have
+been the case when no cloud provider was being used) to using the AWS
+`*instance-id*` (which is what the AWS cloud provider specifies).  To resolve
+this issue:
+
+.  Log in as cluster administrator.
```

Comment:
Specifically I think this is supposed to mean "Login to the CLI as a cluster administrator."